Update manual for MySQL, Xenial support drop (#1501)
theforeman/theforeman.org: _includes/manuals/nightly/3.4_install_from_source.md

Foreman will run with the following requirements (aside from rubygem dependencies):

* Ruby 2.3 or newer
* NodeJS 10 or newer
* NPM 3.0 or newer

> You will want to make sure that you have one of the 

Reviewer: ```suggestion
You will want to make sure that you have either
```
